pkg/platform/model/svc.go

```diff
@@ -85,10 +85,15 @@ func (m *SvcModel) CheckUpdate(ctx context.Context) (*graph.AvailableUpdate, err
 	return &u.AvailableUpdate, nil
 }
 
-func (m *SvcModel) StopServer() error {
+func StopServer(cfg *config.Instance) error {
```

Comment:
I don't think this should be on the model at all. This was only called from one place right? Just implement the logic there. Or if it needs to stay in the model; name it something more explicit. `model.StopServer()`; what server? Probably want `model.StopStateService()` or something. But I'd rather not see it on the model at all.

Reply:
I've moved the function to the service manager that called it originally. Though if we ever do have a need to stop the server from the model we will have to move the code back.